src/tim/vx/tensor.cc

```diff
@@ -181,6 +189,12 @@ bool TensorImpl::Init() {
 
   PackTensorDtype(spec_, &attr.dtype);
 
+  if (!fd_){
+    attr.vsi_memory_type = VSI_MEMORY_TYPE_DMABUF;
+    id_ = vsi_nn_AddTensorFromHandle(graph_->graph(), VSI_NN_TENSOR_ID_AUTO,
+                                     &attr, (uint8_t*)fd_);
+  }
+
```

Comment:
not correct. if we already create tensor from DMA handle, why we still create it another tensor later?